phpunit.xml.dist

```diff
@@ -112,6 +112,10 @@
 		<exclude>tests/Backend/Mixed/Workspaces/S3WorkspacesTest.php</exclude>
 	</testsuite>
 
+	<testsuite name="backend-mixed-synapse">
```

Comment:
Vytvor jeste set env template.

Reply:
`set-env.mixed.template.sh` platí aj pro tůto suite.